rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/implicits.scala

```diff
@@ -0,0 +1,42 @@
+package coop.rchain.rholang.interpreter.storage
+
+import cats.implicits._
+import coop.rchain.models.{Channel, Par}
+import coop.rchain.models.Channel.ChannelInstance.Quote
+import coop.rchain.storage.{Match => StorageMatch}
+import coop.rchain.rholang.interpreter.SpatialMatcher._
+
+import scala.annotation.tailrec
+
+//noinspection ConvertExpressionToSAM
+object implicits {
+
+  implicit def orderingIntTuple[T]: Ordering[(Int, T)] =
```

Comment:
You can use the `Ordering.by` method:
```
def orderingIntTuple[T]: Ordering[(Int, T)] = Ordering.by[(Int, T), Int](_._1)

```

Reply:
Thanks for pointing that out!  I had forgotten about `Ordering.by`...